evaluation/benchmarks/swe_bench/loc_eval/scripts/run_eval_loc.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+# Config
+INFER_DIR="./llm.claude-3-5-haiku1/litellm_proxy_claude-3-5-haiku-20241022"
+EVAL_DIR="./evaluation/evaluation_outputs/outputs/princeton-nlp__SWE-bench_Verified-test/CodeActAgent/claude-3-5-haiku-20241022_maxiter_20_N_v0.38.0-no-hint-run_1/eval_outputs"
+OUTPUT_DIR="./evaluation/benchmarks/swe_bench/loc_eval/eval_saves"
+SPLIT="test"
+DATASET="princeton-nlp/SWE-bench_Verified"
+MAX_TURN=20
```

Comment:
Please don't hard-code anything (providing default is ok though - similar to what we are doing in swe_bench/run_infer.sh), instead, you should document the usage of this script in README.md